subsys/bluetooth/controller/ll_sw/ctrl.c

```diff
@@ -8294,7 +8301,9 @@ static inline int event_len_prep(struct connection *conn)
 		} else if (conn->llcp_feature.features &
 			   BIT(BT_LE_FEAT_BIT_PHY_2M)) {
 			lr->max_rx_time =
-				RADIO_PKT_TIME(LL_LENGTH_OCTETS_RX_MAX, BIT(1));
+				MAX(RADIO_PKT_TIME(LL_LENGTH_OCTETS_RX_MAX,
```

Comment:
This seems to be the correct way of setting max_rx_time, and should be copied to the other places where we set max_rx_time as well

I am doubtfull that this will fix the issue we've seen (receiving incorrect rx-times when executing DLE procedure), and we should run some tests with this fix